src/bun.js/bindings/ModuleLoader.cpp

```diff
@@ -80,14 +77,14 @@ generateInternalModuleSourceCode(JSC::JSGlobalObject* globalObject, InternalModu
         if (!object) {
             return;
         }
-        RETURN_IF_EXCEPTION(throwScope, {});
+        RETURN_IF_EXCEPTION(throwScope, void());
```

Comment:
{} doesn't coerce to jsUndefined anymore?

Reply:
the function needs to return void